src/Tests/AuthWebTestCase.php

```diff
@@ -64,7 +64,10 @@ protected function tearDown(): void
         unset($this->client);
     }
 
-    protected function login(bool $admin = true): void
+    /**
+     * @param string[] $roles
+     */
+    protected function login($roles = ['ROLE_ADMIN']): void
```

Comment:
I know this was the old behavior, but would it not make more sense to have `ROLE_USER` as the default. And pass `ROLE_ADMIN` if you want to elevate your privileges. 

I feel this would more accurately mimic the real life situation. (default is user, admin requires an action)

Reply:
Although I sympathize with this reasoning, most of the current tests currently assume admin privileges. This might result in a lot of updates throughout the test code. Personally, I feel that's outside of the scope of this PR.

Reply:
how about we merge this as is, and make an issue to refactor this in the future?

Reply:
Works for me!